Content.Shared/VendingMachines/Components/VendingMachineVisualStateComponent.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Robust.Shared.GameStates;
+
+namespace Content.Shared.VendingMachines.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed class VendingMachineVisualStateComponent: Component
+{
+    /// <summary>
+    /// RSI state for when the vending machine is unpowered.
+    /// Will be displayed on the layer <see cref="VendingMachineVisualLayers.Base"/>
+    /// </summary>
+    [DataField("offState")]
+    public string? OffState;
+
+    /// <summary>
+    /// RSI state for the screen of the vending machine
+    /// Will be displayed on the layer <see cref="VendingMachineVisualLayers.Screen"/>
+    /// </summary>
+    [DataField("screenState")]
+    public string? ScreenState;
+
+    /// <summary>
+    /// RSI state for the vending machine's normal state. Usually a looping animation.
+    /// Will be displayed on the layer <see cref="VendingMachineVisualLayers.BaseUnshaded"/>
+    /// </summary>
+    [DataField("normalState")]
+    public string? NormalState;
+
+    /// <summary>
+    /// RSI state for the vending machine's eject animation.
+    /// Will be displayed on the layer <see cref="VendingMachineVisualLayers.BaseUnshaded"/>
+    /// </summary>
+    [DataField("ejectState")]
+    public string? EjectState;
+
+    /// <summary>
+    /// RSI state for the vending machine's deny animation. Will either be played once as sprite flick
+    /// or looped depending on how <see cref="LoopDenyAnimation"/> is set.
+    /// Will be displayed on the layer <see cref="VendingMachineVisualLayers.BaseUnshaded"/>
+    /// </summary>
+    [DataField("denyState")]
+    public string? DenyState;
+
+    /// <summary>
+    /// RSI state for when the vending machine is unpowered.
```

Comment:
This is same summary as `OffState`? Shouldn't it say broken instead of unpowered?

Reply:
> This is same summary as `OffState`? Shouldn't it say broken instead of unpowered?

I didn't notice. I'll fix it